### ISSUE TITLE
[$15]"After ""Number of Registrants"" link, there is a link with link txt as ""0"".  Provide proper link txt. "

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3241]
**Describe the bug**
After ""Number of Registrants"" link, there is a link with link txt as ""0"".  Provide proper link txt.

**To Reproduce**
1. Go to the Challenges page: https://www.topcoder.com/challenges. 
2. Ensure the screen readr is switched on. 
3. Tab to a challenge 
4. Tab till you get to the Submission #, which is after the ""Number of registrants"" link.

**Expected behavior**

Should say "Total Submitters: # Link"

**Actual behavior**
Screenreader reads "# Visited Link" or "Zero Link".

**Additional context**

Type: Challenge Listing -> Challenge Information

Page: https://www.topcoder.com/challenges

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099938 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099938 has been updated - it has been assigned to nahidshahin.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by nahidshahin** [comment]
@veshu the problem already solved at `qa-accessibility` branch but not in `develop` branch.

The fix is at this line: 
https://github.com/topcoder-platform/community-app/blob/qa-accessibility/src/shared/components/challenge-listing/ChallengeCard/NumSubmissions/index.jsx#L54

**Comment by veshu** [comment]
Yes the code is on branch.

@sushilshinde looks code is not moved on dev and master.

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099938<br/><br/>```This is an automated message for crazyk via Topcoder X```
